docs-ref-conceptual/what-is-azure-cli.md

```diff
@@ -1,40 +1,60 @@
 ---
-title: What is Azure CLI?
-description: Overview of Azure CLI, a command-line interface tool designed to create and manage Azure resources now available in Windows, macOS, and Linux environments.
+title: What is the Azure CLI?
+description: Overview of the Azure CLI, a command-line tool designed to create and manage Azure resources available in Windows, macOS, Linux, and Docker containers.
 author: dbradish-microsoft
 ms.author: dbradish
 manager: barbkess
-ms.date: 01/30/2020
+ms.date: 06/04/2021
 ms.topic: overview
 ms.service: azure-cli
 ms.devlang: azurecli 
 ms.custom: devx-track-azurecli
 ---
-# What is Azure CLI
+# What is the Azure CLI
```

Comment:
Please use a question mark here: What is the Azure CLI? 
It's a nit, I know. :)